### ISSUE TITLE
toml-action breaks on "foo.bar = 123" syntax

### ISSUE THREAD

**Issue by ajeetdsouza** [SebRollen/toml-action#5]
TOML supports the following syntax:

```toml
foo.bar = 123
```

which is equivalent to

```toml
foo = { bar = 123 }
```

However, toml-action throws an error when it encounters this:

```
Expected "=", [ \t] or [A-Za-z0-9_\-] but "." found.
```

**Comment by SebRollen** [comment]
This seems to be an issue in the underlying toml library that this action uses: https://github.com/BinaryMuse/toml-node/issues/51

Seems like other libraries have had success with switching to `@iarna/toml` - I'll try the same

**Comment by SebRollen** [comment]
This is now fixed in version 1.0.2 https://github.com/SebRollen/toml-action/releases/tag/v1.0.2

Thanks for opening the issue!
